setup.py

```diff
@@ -250,7 +250,7 @@ def get_exts_for(name):
         "protobuf>=3,<4.0; python_version=='3.6'",
         "protobuf>=3,<3.18; python_version<'3.6'",
         "tenacity>=5",
-        "attrs>=19.2.0",
+        "attrs>=21.4.0",
```

Comment:
Why was this change necessary? Should we include an `upgrade` release note that our minimum version support for `attrs` changed?

Reply:
@Kyle-Verhoog CI started to fail with this error:

```
  File "/root/project/.riot/venv_py3103/lib/python3.10/site-packages/cattr/disambiguators.py", line 15, in <module>
    from cattrs._compat import get_origin
  File "/root/project/ddtrace/internal/module.py", line 190, in _exec_module
    self.loader.exec_module(module)
  File "/root/project/.riot/venv_py3103/lib/python3.10/site-packages/cattrs/_compat.py", line 16, in <module>
    from attr import resolve_types
ImportError: cannot import name 'resolve_types' from 'attr'
```

https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/21452/workflows/b043126a-d4c8-49d3-84cc-703e95225486/jobs/1460754

Reply:
Is there an older version of attrs we can use (preferably the oldest)? We should be as unrestrictive as possible with our dependency versions.

Also we should still have an `upgrade` note since this could be a breaking change for users.